rotkehlchen/db/dbhandler.py

```diff
@@ -1191,6 +1191,64 @@ def edit_blockchain_accounts(
             object_reference_keys=['chain', 'address'],
         )
 
+    def edit_chain_type_accounts_labels(
+            self,
+            write_cursor: 'DBCursor',
+            account_data: list[SingleBlockchainAccountData],
+            address_to_blockchains: dict[str, list[SupportedBlockchain]],
+    ) -> None:
+        """Edit the given accounts for all the blockchains where they are tracked
+
+        At this point in the calling chain we should already know that:
+        - All tags exist in the DB
+        """
+        # Update the blockchain account labels in the DB
+        write_cursor.executemany(
+            'DELETE FROM address_book WHERE address=?',
+            [(account.address,) for account in account_data],
+        )
+        write_cursor.executemany(
```

Comment:
We need changes here. This makes no sense.

There should be only a **single entry** for all chains. Seems you are iterating stuff here.